Modules/clinic/_sre.c.h

```diff
@@ -690,7 +690,7 @@ PyDoc_STRVAR(_sre_SRE_Match_span__doc__,
 "span($self, group=0, /)\n"
 "--\n"
 "\n"
-"For MatchObject m, return the 2-tuple (m.start(group), m.end(group)).");
+"For match object m, return the 2-tuple (m.start(group), m.end(group)).");
```

Comment:
I don't know. The term "match object" is widely used in the documentation (unlikely to just added name `Match`).